src/test/java/com/wikia/webdriver/common/clicktracking/events/EventsArticleEditMode.java

```diff
@@ -17,8 +17,7 @@
       .add(
           "1",
           Json.createObjectBuilder().add(EventParameter.LABEL.getEventParameter(),
-                                         "preview-desktop")).build();
+              "preview-desktop")).build();
 
-  private EventsArticleEditMode() {
-  }
+  private EventsArticleEditMode() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)